<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>976</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ELF Detective</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>580</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QCheckBox" name="checkHex">
            <property name="toolTip">
             <string>See intruction hexcodes</string>
            </property>
            <property name="text">
             <string>Hexcodes</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="addExe">
            <property name="text">
             <string>Add Executable</string>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignRight">
           <widget class="QPushButton" name="addObj">
            <property name="text">
             <string>Add Object</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="runProj">
            <property name="text">
             <string>Run</string>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignRight">
           <widget class="QPushButton" name="clearProj">
            <property name="text">
             <string>Clear Project</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QToolBox" name="exeToolBox">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="exeDataPage">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>474</width>
             <height>319</height>
            </rect>
           </property>
           <attribute name="label">
            <string>Page 1</string>
           </attribute>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QListWidget" name="exeDataList"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="exeFunctionPage">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>474</width>
             <height>319</height>
            </rect>
           </property>
           <attribute name="label">
            <string>Page 2</string>
           </attribute>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QTreeView" name="exeFunctionsTree"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QTabWidget" name="objTabs">
          <widget class="QWidget" name="tab">
           <attribute name="title">
            <string>Tab 1</string>
           </attribute>
           <layout class="QHBoxLayout" name="horizontalLayout_5"/>
          </widget>
          <widget class="QWidget" name="tab_2">
           <attribute name="title">
            <string>Tab 2</string>
           </attribute>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item alignment="Qt::AlignBottom">
       <widget class="QTextBrowser" name="infoOutput"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
